test/src/databases/presto/presto.spec.ts

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ */
+
+/* eslint-disable no-console */
+
+import {RuntimeList} from '../../runtimes';
+import {describeIfDatabaseAvailable} from '../../util';
+import '../../util/db-jest-matchers';
+
+const [describe] = describeIfDatabaseAvailable(['presto']);
```

Comment:
Since you're adding these functions to both Presto and Trino, these tests should run against both. Probably rename this file `presto-trino/presto-trino.spec.ts`, and change this line to  
```ts
const [describe, databases] = describeIfDatabaseAvailable(['presto', 'trino']);
```

and the beginning of the tests to

```ts
const runtimes = new RuntimeList(databases);
describe.each(runtimes.runtimeList)(
  'Presto/Trino dialect functions - %s',
  (databaseName, runtime) => {
```